grails-app/controllers/org/pih/warehouse/JsonController.groovy

```diff
@@ -93,8 +98,31 @@ class JsonController {
                 requisition.updatedBy = session.user
                 requisition.addToRequisitionItems(requisitionItem)
                 if (requisition.validate() && requisition.save(flush: true)) {
-                    json = [success: true, data: requisition]
+                    json = [success: true, data: [
```

Comment:
This needs to be more DRY. Move to the data map to a method in Requisition ... call it something like toStockListDetailsJson() or something like that.